package/crossplane.yaml

```diff
@@ -1,16 +1,25 @@
 apiVersion: meta.pkg.crossplane.io/v1alpha1
 kind: Provider
 metadata:
-  name: provider-aws
+  name: provider-aws-{{ APIGROUP }}
+  labels:
+    pkg.crossplane.io/provider-family: aws
   annotations:
     meta.crossplane.io/maintainer: Upbound <support@upbound.io>
     meta.crossplane.io/source: github.com/upbound/provider-aws
     meta.crossplane.io/description: |
       Upbound's official Crossplane provider to manage Amazon Web Services (AWS)
-      resources in Kubernetes.
+      {{ APIGROUP }} services in Kubernetes.
     meta.crossplane.io/readme: |
       Provider AWS is a Crossplane provider for [Amazon Web Services
       (AWS)](https://aws.amazon.com/) developed and supported by Upbound.
-      If you encounter an issue please reach out on our support@upbound.io
-      email address.
+      Available resources and their fields can be found in the [Upbound
+      Marketplace](https://marketplace.upbound.io/providers/upbound/provider-aws).
+      If you encounter an issue please reach out on support@upbound.io email
+      address. This is a subpackage for the {{ APIGROUP }} API group.
+    friendly-name.meta.crossplane.io: Provider AWS ({{ APIGROUP }})
     auth.upbound.io/group: aws.upbound.io
+spec:
+  dependsOn:
+    - provider: {{ XPKG_REG_ORGS }}/provider-aws-config
+      version: "*"
```

Comment:
Wouldn't this bring and rc version as a dependency to a released version? Which is why I suggested `>=v0.0.0` in the first place.